modules/network/vpc/README.md

```diff
@@ -1,20 +1,28 @@
 ## Description
 
-This module creates a new VPC network along with a
-[cloud NAT](https://github.com/terraform-google-modules/terraform-google-cloud-nat),
-[Router](https://github.com/terraform-google-modules/terraform-google-cloud-router)
-and common [firewall rules](https://github.com/terraform-google-modules/terraform-google-network/tree/master/modules/firewall-rules).
-This module is based on submodules defined by the
-[Cloud Foundation Toolkit](https://cloud.google.com/foundation-toolkit).
-
-The created cloud NAT (Network Address Translation) allows virtual machines
-without external IP addresses to create outbound connections to the internet.
-For more information see the [docs](https://cloud.google.com/nat/docs/overview).
-
-The following firewall rules are created with the VPC network:
-
-* Allow SSH access from the Cloud Console ("35.235.240.0/20").
-* Allow traffic between nodes within the VPC
+This module creates a new [VPC network][vpc], subnetworks, and [Cloud
+Router][router]. By default, it will enable
+
+* [Cloud NAT][nat] to enable outbound access to the public internet for all VMs
+* [Firewall rule][fwrules] that enables inbound SSH access from [Identity-Aware
+  Proxy][iap]
+* [Firewall rule][fwrules] that enables all traffic internal to the network
+
+This behavior is optional and can be configured as [described below](#inputs).
+This module is based on networking support in the [Cloud Foundation
+Toolkit][cft]. We recommend following the [documentation for the network
+module][cft-network] and [submodules][cft-network-submodules] for detailed
```

Comment:
nit: The links here take a little poking around to find what you are referencing. 

The `cft-network` link has the `secondary_ranges` definition. To get the `firewall_rules` definition I go to `cft-network-submodules` link and then click on `firewall-rules`.  

Why not just link directly to the firewall-rules module?
Why not put the link next to the respective variable so it is clear what to click on for what?

Reply:
Addressed nit on HEAD of PR with a bulleted list toward top. I think we need a broader strategy for `variables.tf` documentation that doesn't involve clunky multi-line HEREDOCs with extremely long URLs.